src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -512,6 +523,18 @@ public void test() {
         //even with the same name should work
         createStudy("userId2", STUDY_NAME, CASE_UUID, DESCRIPTION, true);
         studyUuid = studyRepository.findByUserIdAndStudyName("userId2", STUDY_NAME).get().getId();
+        UUID oldStudyUuid = studyRepository.findByUserIdAndStudyName("userId", STUDY_NAME).get().getId();
+
+        webTestClient.get()
+                .uri("/v1/studies/metadata")
+                .header("userId", "userId")
+                .header("uuids", oldStudyUuid + "," + studyUuid)
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.APPLICATION_JSON)
+                .expectBodyList(CreatedStudyBasicInfos.class)
+                .value(studies -> studies.get(0),
+                        createMatcherCreatedStudyBasicInfos(oldStudyUuid, STUDY_NAME, "userId", "UCTE", "description", false));
```

Comment:
Maybe find a way to check that both _oldStudyUuid_ and _studyUuid_ studies metadata are well retrieved. Here, we just check the metadata of study _oldStudyUuid_.

Reply:
we should create a matcher that accept a list in a separate PR